I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 541, in script
    jump presentation2
ScriptError: could not find label 'presentation2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 541, in script
    jump presentation2
  File "/home/carlos/renpy-7.1.3-sdk/renpy/ast.py", line 1633, in execute
    rv = renpy.game.script.lookup(target)
  File "/home/carlos/renpy-7.1.3-sdk/renpy/script.py", line 875, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'presentation2'.

Linux-4.18.0-12-generic-x86_64-with-debian-buster-sid
Ren'Py 7.1.3.1092
The Open Source Club 1.0
Fri Dec 14 14:00:33 2018
